packages/lib-react-components/src/ZooHeader/ZooHeader.js

```diff
@@ -55,6 +55,7 @@ export default function ZooHeader (props) {
 
   return (
     <StyledHeader
+      aria-label={counterpart('ZooHeader.ariaLabel')}
```

Comment:
I'll double-check, but we probably don't need to label these headers as long as the page heading structure is clearly marked up.